README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+
+### L. E. Hunter - Biography
+
+Hello, everyone! My name is Laura. I'm an incoming **Integrative Biology** student interested in human evolution. I graduated from Columbia University in 2017 and have been working at a [neuroscience lab](https://www.gottlieblab.com/) at the [Zuckerman Institute for Mind, Brain, and Behavior](https://zuckermaninstitute.columbia.edu/) ever since. In my spare time, I work as an emergency medical technician and volunteer as a fossil and primatology explainer at the American Museum of Natural History (although the museum stuff has been on hold since March due to COVID). I also like to paint. Check out my painting of the skeleton of the animal I studied while working at the Zuckerman Institute:
+
+![Image of Macaque Skeleton](https://user-images.githubusercontent.com/12615998/86480299-bd8b2180-bd1b-11ea-921e-3f4c33582e7f.png)
```

Comment:
I love that you included an image and the code appears corrects however it does not show up here unless you also commit the image file here. (At least I believe thats how it works, IF I did my image correctly lol). I was able to plug it into the web browser and it showed up though so may be wrong. Including the image was the hardest part for me. Either way I was able to see your image, awesome painting!